run/image-processing/e2e_test.py

```diff
@@ -231,7 +231,7 @@ def test_end_to_end(input_bucket, output_bucket):
     # Wait for image processing to complete
     time.sleep(60)
 
-    for x in range(10):
+    for x in range(20):
```

Comment:
Instead of just increasing the time, could you try setting a [retry](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#retry-rpcs) if it can be used here?

Reply:
the RPC does not fail, it just returns empty results until processing completes.
While we could do this with a linear backoff, I think the resulting code would actually be less readable, as it would require an extra function or inlining the check as a predicate.
What do you see as the benefits of using a retry here?

Reply:
I'd say it scales _somewhat_ better and is more reliable than simply retrying x more times, if we're not sure why it's failing and being flaky. However, in this case it seems like trying with retries may not be the most suitable as this .. surprisingly doesn't fail or throw exceptions.